doc/specs/Keyboard-Selection.md

```diff
@@ -0,0 +1,143 @@
+---
+author: Carlos Zamora @cazamor
+created on: 2019-08-30
+last updated: 2019-09-21
+issue id: #715
+---
+
+# Keyboard Selection
+
+## Abstract
+
+This spec describes a new set of keybindings that allows the user to create and update a selection without the use of a mouse or stylus.
+
+## Inspiration
+
+ConHost allows the user to edit an existing selection using the keyboard. Holding `Shift` allows the user to move the second selection anchor in accordance with the arrow keys. The selection anchor updates by one cell per key event, allowing the user to refine the selected region. However, mouse selection seems to be superior in that it provides access to the following special selection modes:

```

Comment:
I might take out the part about "However, mouse selection seems to be superior". I think this is supposed to be a spec selling me on why a user would want mark mode, not "lol just use the mouse, it's better"